src/executors/public_1559_executor.rs

```diff
@@ -241,6 +227,63 @@ where
                 info!("{} - Failed to release key: {}", order_hash, e);
             }
         }
+
+        // post key-release processing
+        // TODO: parse revert reason
+        if let Some(cw) = &self.cloudwatch_client {
+            let metric_future = cw
+                .put_metric_data()
+                .namespace(ARTEMIS_NAMESPACE)
+                .metric_data(
+                    MetricBuilder::new(receipt_status_to_metric(status.as_u64()))
+                        .add_dimension(
+                            DimensionName::Executor.as_ref(),
+                            DimensionValue::PriorityExecutor.as_ref(),
+                        )
+                        .with_value(1.0)
+                        .build(),
+                )
+                .send();
+
+            send_metric_with_order_hash!(&order_hash, metric_future);
+        }
+
+        if let Some(TransactionReceipt {
+            block_number: Some(block_number),
+            ..
+        }) = receipt
+        {
+            // log wallet balance every 100 blocks
+            if block_number.as_u64() % 100 == 0 {
+                let balance_eth = self
+                    .client
+                    .get_balance(address, Some(block_number.into()))
+                    .await
+                    .map_or_else(|_| None, |v| Some(format_units(v, "ether").unwrap()));
+                
+                // TODO: use if-let chains when it becomes stable https://github.com/rust-lang/rust/issues/53667
+                // if let Some(balance_eth) = balance_eth && let Some(cw) = &self.cloudwatch_client {
+                if let Some(balance_eth) = balance_eth {
+                    info!("{}- balance: {} at block {}", order_hash, balance_eth.clone(), block_number.as_u64());
+                    if let Some(cw) = &self.cloudwatch_client {
+                        cw.put_metric_data()
+                            .namespace(ARTEMIS_NAMESPACE)
+                            .metric_data(
+                                MetricBuilder::new(CwMetrics::Balance(address.to_string()))
+                                    .add_dimension(
+                                        DimensionName::Executor.as_ref(),
+                                        DimensionValue::PriorityExecutor.as_ref(),
+                                    )
```

Comment:
Should just be one not both dimensions right?

Reply:
one is name and the other is value! But yeah, honestly don't need such granularity